week5/5.1/that is the way.py

```diff
@@ -0,0 +1,27 @@
+import os
+"""
+That's the way.
+By: Eliad Karni.
+
+The code receive a file path from the user and prints a list of the file in the received directory,
+which their names starts with PREFIX value.
+
+used websites help:
```

Comment:
1. Remember to use capital letters.
2. It should be written differently (grammer wise).